### DISCUSSION TITLE
[v-line] config fill option doesn't work!

### DISCUSSION THREAD

**Discussion by Mt-Youya** [konvajs/konva#1698] [Q&A] [answered]
I'm so sorry , my English is bad!

I use v-line in vue@3.3.9 & vite@5.0.4 , but fill option doesn't work!

[code here](https://codesandbox.io/p/devbox/youthful-ishizaka-3k43nl?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clqyxdvmw0007356ig295ayia%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clqyxdvmw0002356iz8vaoslg%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clqyxdvmw0004356iy931a895%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clqyxdvmw0006356ieb0x5qa4%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clqyxdvmw0002356iz8vaoslg%2522%253A%257B%2522id%2522%253A%2522clqyxdvmw0002356iz8vaoslg%2522%252C%2522tabs%2522%253A%255B%255D%257D%252C%2522clqyxdvmw0006356ieb0x5qa4%2522%253A%257B%2522id%2522%253A%2522clqyxdvmw0006356ieb0x5qa4%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clqyxdvmw0005356ix0ytis0q%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522npm%2520run%2520dev%2522%252C%2522port%2522%253A5173%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clqyxdvmw0005356ix0ytis0q%2522%257D%252C%2522clqyxdvmw0004356iy931a895%2522%253A%257B%2522id%2522%253A%2522clqyxdvmw0004356iy931a895%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clqyxdvmw0003356irv81z5z0%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522npm%2520run%2520dev%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clqyxdvmw0003356irv81z5z0%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D)

When i use stroke and  strokeWidth , it works! 

I need use fill line color, and padding line ,cuz when strokeWidth thin , cursor cant catch it , so i need padding in v-line.

I need width 800 , height 4  on the line, but when cursor near line 5px, the line need change color, i think the line fiil color, and strokeWidth is padding , and stroke color is transparent can do it !  But it doesn't work!

**Comment by lavrton** [comment, marked as answer]
If you want to increase hit area for mouse events you can use `hitStrokeWidth` property.
